test/units/module_utils/common/text/formatters/test_human_to_bytes.py

```diff
@@ -79,3 +79,95 @@ def test_human_to_bytes_wrong_number(test_input):
     """Test of human_to_bytes function, nubmer param is invalid string / number."""
     with pytest.raises(ValueError, match="can't interpret"):
         human_to_bytes(test_input)
+
+
+@pytest.mark.parametrize(
+    'input_data,expected',
+    [
+        (0, 0),
+        (1024, 1024),
+        (u'1024b', 1024),
+        (u'1024B', 1024),
+        (u'0B', 0),
+        (u'1K', 1024),
+        (u'1Kb', 1024),
+        (u'1M', 1048576),
+        (u'1Mb', 1048576),
+        (u'1G', 1073741824),
+        (u'1Gb', 1073741824),
+        (u'1T', 1099511627776),
+        (u'1Tb', 1099511627776),
+        (u'1P', 1125899906842624),
+        (u'1Pb', 1125899906842624),
+        (u'1E', 1152921504606846976),
+        (u'1Eb', 1152921504606846976),
+        (u'1Z', 1180591620717411303424),
+        (u'1Zb', 1180591620717411303424),
+        (u'1Y', 1208925819614629174706176),
+        (u'1Yb', 1208925819614629174706176),
+    ]
+)
+def test_human_to_bytes_isbits(input_data, expected):
+    """Test of human_to_bytes function, isbits = True."""
+    assert human_to_bytes(input_data, isbits=True) == expected
+
+
+@pytest.mark.parametrize(
+    'input_data,unit,expected',
+    [
+        (1024, u'b', 1024),
+        (u'1024', u'B', 1024),
+        (1, u'K', 1024),
+        (1, u'Kb', 1024),
+        (u'1', u'M', 1048576),
+        (u'1', u'Mb', 1048576),
+        (1, u'G', 1073741824),
+        (1, u'Gb', 1073741824),
+        (1, u'T', 1099511627776),
+        (1, u'Tb', 1099511627776),
+        (u'1', u'P', 1125899906842624),
+        (u'1', u'Pb', 1125899906842624),
+        (u'1', u'E', 1152921504606846976),
+        (u'1', u'Eb', 1152921504606846976),
+        (u'1', u'Z', 1180591620717411303424),
+        (u'1', u'Zb', 1180591620717411303424),
+        (u'1', u'Y', 1208925819614629174706176),
+        (u'1', u'Yb', 1208925819614629174706176),
```

Comment:
You copy-paste a lot. It'd be more resilient if you could just put repeatable parts of data sets into reusable vars.

Reply:
@webknjaz , in general I completely agree with you.
However, in this case we have different test tuples for every test, for exemple:
```
 21         (u'1KB', 1024)
 48         (1, u'KB', 1024),
 93         (u'1Kb', 1024),
121         (1, u'Kb', 1024),
```
Or maybe you mean something other?

Reply:
There's subset of cases which are the same, you could put those into a variable and then merge that with extra cases on per-test basis

Reply:
@webknjaz PTAL

Reply:
IMO now it looks better :)